README.md

```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images hosted on Docker Hub. The main tenets are:
+
+- Support multiple architectures
 
-## What do you mean by "Official"?
+- Exemplify Dockerfile best practices
 
-In many cases, the images in this program are not only supported but also *maintained* directly by the relevant upstream projects.
+- Actively rebuilt for updates and security fixes
```

Comment:
```suggestion
- [Actively rebuilt](#library-definition-files) for updates and security fixes
```

(Hopefully this one will have a better place to link at some point, but for now this is the most "canonical" location.)